.github/workflows/regression-report.yml

```diff
@@ -56,8 +63,22 @@ jobs:
               content: 'rocket'
             })
 
-      - name: Checkout repository
-        uses: actions/checkout@v4
```

Comment:
Restore this, we need to checkout right before it is required, not at the beginning